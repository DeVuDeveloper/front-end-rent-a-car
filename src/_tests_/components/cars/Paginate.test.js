import React from 'react';
import { Provider } from 'react-redux';
import { BrowserRouter as Router } from 'react-router-dom';
import renderer from 'react-test-renderer';
import Paginate from '../../../components/cars/Paginate';
import store from '../../../redux/configureStore';

it('should render Paginate component correctly', () => {
  const tree = renderer.create(
    <Provider store={store}>
      <Router>
        <Paginate />
      </Router>
    </Provider>,
  );
  expect(tree).toMatchSnapshot();
});
