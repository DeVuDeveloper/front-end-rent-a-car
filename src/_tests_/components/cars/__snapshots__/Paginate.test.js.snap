// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Paginate component correctly 1`] = `
<ul>
  <li
    className="previous disabled"
  >
    <a
      aria-disabled="true"
      aria-label="Previous page"
      className="previous-btn disable-btns"
      onClick={[Function]}
      onKeyPress={[Function]}
      rel="prev"
      role="button"
      tabIndex="-1"
    >
      ᐊ
    </a>
  </li>
  <li
    className="next"
  >
    <a
      aria-disabled="false"
      aria-label="Next page"
      className="next-btn"
      onClick={[Function]}
      onKeyPress={[Function]}
      rel="next"
      role="button"
      tabIndex="0"
    >
      ᐅ
    </a>
  </li>
</ul>
`;
